traffic_ops/traffic_ops_golang/server/servers_server_capability.go

```diff
@@ -121,7 +121,7 @@ func (ssc TOServerServerCapability) Validate() error {
 	return util.JoinErrs(tovalidate.ToErrors(errs))
 }
 
-func (ssc *TOServerServerCapability) Update() (error, error, int) {
+func (ssc *TOServerServerCapability) Update(h http.Header) (error, error, int) {
```

Comment:
I know you didn't create this function, but it can be deleted.

This is a legacy from when the "CRUDer" stuff required every func be implemented. This endpoint doesn't have a PUT method, and the "CRUDer" framework no longer requires all funcs, so it can just be deleted now.